contact.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - About</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+
+<body>
+  <div class="container contact-page">
+    <header>
+  
+      <nav>
+        <img src="img/lambda-black.png" alt="lambda logo">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="contact.html">Contact</a>
+      </nav>
+      <img src="img/jumbo-contact.jpg" alt="handshake" class="contact-header-img">
+    </header>
+
+    <section class="contact-main">
+        <h2>Contact Form</h2>
+        <form action="thankyou.html">
+          First name: <input type="text" name="fname"><br>
+          Last name: <input type="text" name="lname"><br>
+          E-Mail: <input type="text" name="e-mail"><br>
```

Comment:
Check out the different type attributes to use with `<input>`. For instance, you can use `type="email"` for some additional features.